include/cvc5/cvc5_proof_rule.h

```diff
@@ -2481,6 +2481,93 @@ enum ENUM(ProofRewriteRule) : uint32_t
    * \endverbatim
    */
   EVALUE(DT_CONS_EQ),
+
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Bitvectors - Unsigned multiplication overflow detection elimination**
+   *
+   * See M.Gok, M.J. Schulte, P.I. Balzola, "Efficient integer multiplication
+   * overflow detection circuits", 2001.
+   * http://ieeexplore.ieee.org/document/987767
+   * \endverbatim
+   */
+  EVALUE(BV_UMULO_ELIMINATE),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Bitvectors - Signed multiplication overflow detection elimination**
+   *
+   * See M.Gok, M.J. Schulte, P.I. Balzola, "Efficient integer multiplication
+   * overflow detection circuits", 2001.
+   * http://ieeexplore.ieee.org/document/987767
+   * \endverbatim
+   */
+  EVALUE(BV_SMULO_ELIMINATE),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Bitvectors - Flatten nested add/or/mult/xor/and expression**
+   *
+   * .. math::
+   *    ((a1 and a2) and (a3 and a4)) \to (a1 and a2 and a3 and a4)
+   *
+   * \endverbatim
+   */
+  EVALUE(BV_FLATTEN_ASSOC_COMMUT),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Bitvectors - Flatten nested add/or/mult/xor/and expression**
+   *
+   * .. math::
+   *    ((a1 and a2) and (a3 and a4)) \to (a1 and a2 and a3 and a4)
+   *
+   * \endverbatim
+   */
+  EVALUE(BV_FLATTEN_ASSOC_COMMUT_NO_DUPLICATES),
```

Comment:
`ACI_NORM`?

Reply:
Indeed, this may be subsumed by ACI_NORM now.

Reply:
deleted